### DISCUSSION TITLE
Custom Footer in Login and Dashboard

### DISCUSSION THREAD

**Discussion by jcgmalave** [filamentphp/filament#13686] [Help]
### Package

Panel builder

### Package Version

3.0.0

### How can we help you?

how can i show the custom footer view footer.blade.php in the Login Page and in the Dashboard, i already apply to all of the resources pages; thanks in advanced.

**Comment by leandrocfe** [comment]
try render hooks
https://filamentphp.com/docs/3.x/support/render-hooks

**Reply by jcgmalave** [reply]
i get this error: Class "Filament\View\PanelsRenderHook" not found

**Reply by leandrocfe** [reply]
`php artisan about --only=Filament`
What is the Filament version?
